lib/index.d.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { NavigationRouteConfigMap, StackNavigatorConfig } from 'react-navigation';
+
+export { RouteDirection, Metrics, TransitionSpecification } from './Types';
+
+export function createFluidNavigator(
+  routeConfigMap: NavigationRouteConfigMap,
+  stackConfig: StackNavigatorConfig = {}
+): any;
```

Comment:
```suggestion
): NavigationContainer;
```
as similar as in https://github.com/react-navigation/react-navigation/blob/542122794ad36d9ae68f33a453e31563147280bb/typescript/react-navigation.d.ts#L938